seqr/views/utils/permissions_utils.py

```diff
@@ -228,24 +228,25 @@ def check_multi_project_permissions(obj, user):
 
 
 def get_project_guids_user_can_view(user, limit_data_manager=True):
+    is_data_manager = user_is_data_manager(user)
+    if is_data_manager and not limit_data_manager:
```

Comment:
Is it neater to inline `user_is_data_manager` into the `if` condition and remove `is_data_manager` variable.